### ISSUE TITLE
Possible bug with long-ish arrays

### ISSUE THREAD

**Issue by jamesplease** [reduxjs/redux-devtools#393]
There may be a bug with long arrays dispatched in actions. I dispatched an action with an array of length ~350,000, and the action does not appear in the list of dispatched actions. Restricting the length to 10, 100, 1000, and even 50000 is fine. But going to 100k or beyond makes the action not appear within Redux Devtools.

I know this is an edge-casey thing, but I figured I'd open an issue. Feel free to close if you think that a solution is too difficult/impossible to implement.

Thanks for reading! :v:

//cc @jporry

**Comment by wrakky** [comment]
I've just come across comething similar and it appears to be a consequence of exceeding the maximum size limit chrome extension messages allow which happens when your state becomes excessively large and gives you the following stack trace:
```
Failed to send message Error: Message length exceeded maximum allowed length.
    at PortImpl.postMessage (VM25495 extensions::messaging:121)
    at Port.publicClassPrototype.(:3000/anonymous function) [as postMessage] (extensions::utils:138:26)
    at send (contentScript.js:63)
    at tryCatch (contentScript.js:49)
    at handleMessages (contentScript.js:81)
```

My solution was to use the `actionSanitizer` and `stateSanitizer` options to reduce the size of the offending action and state which then fixed the problem.

It might be an idea to try and handle this issue and display a warning message if possible because in my case the devtools just stopped working with no warning or indication as to why which was quite confusing and wasted a couple of hours of my day investigating the cause.

**Comment by zalmoxisus** [comment]
I think this is related to Chrome extension. https://github.com/zalmoxisus/redux-devtools-extension/pull/582 fixes that and it's available in `2.16`

**Comment by jamesplease** [comment]
Thanks @zalmoxisus !
